providers/summary/clearlydefined.js

```diff
@@ -267,8 +282,18 @@ class ClearlyDescribedSummarizer {
     }
     // We could have singular licenses such as 'MIT' or licenses in an array ['MIT', 'BSD']
     // Process licenses depending on whether they are strings or array of strings
-    if (Array.isArray(manifest.license)) {
-      const licenses = SPDX.normalize((manifest.license || []).join(' OR '))
+    let expression = null  // license expression (as input to SPDX.normalize)
+    if (typeof manifest.license === 'string') {
+      expression = manifest.license
+    } else if (Array.isArray(manifest.license)) {
+      expression = manifest.license.join(' OR ')
+    } else if (typeof manifest.licenses === 'string') {
```

Comment:
@tmarble Can PHP composer have licenses (not license) field? Is it documented anywhere? I wonder if this code will ever execute.